app/assets/stylesheets/config/_import.scss

```diff
@@ -200,6 +229,55 @@ $lh-base: 1.5; // 24px for 16px font-size
 
 ////////////////////////////////////////////////////////////////////////////////////
 
+// initialize-generator()
+//
+// Initilize generators.
+//
+// @param {list} $breakpoints       All our responsiveness breakpoints.
+// @param {list} $classes           All the parameters required by generate-classes().
+
+@mixin initialize-generator($breakpoints, $classes...) {
```

Comment:
I'm not the smartest dev on the earth + SASS has its own limitations but if anyone has ideas to optimize this function, I'm open for suggestions.